ex16_ch4.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: state constant
+LEGAL_DRIVING_AGE = 16
+
+#step2: get input from user on their age
+user_age = int(input("Enter your age: "))
+
+#step3: set up first conditional
+if user_age >= LEGAL_DRIVING_AGE:
+    print("You're old enough to drive legally.")
+
+#step4: set up second conditional if user enters a number less than 0
+if user_age < 0: 
```

Comment:
I'd personally test this before checking the age against the legal driving age. If it isn't a valid age, there's no use in proceeding further.